include/mbedtls/error.h

```diff
@@ -82,6 +82,8 @@
  * POLY1305  3                  0x0057-0x005B
  * CHACHAPOLY 2 0x0054-0x0056
  * PLATFORM  2  0x0070-0x0072
+ * LMOTS     2  0x0080-0x0082
+ * LMS       2  0x0084-0x0088
```

Comment:
Low-level error codes need to be even numbers in the range 0x0002..0x007e.

Reply:
Actually yes, odd numbers are fine. You just need to be careful because odd numbers and even numbers have historically been considered independent ranges, so n..n+2 might belong to one module while n+1 belongs to another module.

This monstrosity prints out the unassigned low-level error codes:
```
comm -13 <(cpp -Iinclude library/error.c |sed -n 's/)://; s/^ * case -(-0x00//p' |sort) <(for x in {2..126}; do printf %02X\\n $x; done)
```
How about 0x0011..0x0019 odds?

Reply:
Yep I'll use those, thanks for the hint